server/bin/import_test_data.rb

```diff
@@ -269,6 +283,9 @@ def create_mkt_product_and_pools(cp, owner, product)
 
   params[:branding] = []
 
+  #Below code is commented because moving branding to product.
```

Comment:
There is no need for this block any more, so you can just remove it instead of commenting out.

Reply:
Sure. I have now removed the commented code.